spec/system/candidate_interface/candidate_signing_in_spec.rb

```diff
@@ -52,4 +52,12 @@
       expect(page).to have_content('Welcome')
     end
   end
+
+  it 'can take the user back to the start page' do
```

Comment:
Let's just Use the third person in the present tense (from betterspec.org)
<img width="684" alt="Screenshot 2019-10-04 at 10 48 22" src="https://user-images.githubusercontent.com/58971/66198577-9a501a80-e694-11e9-91d4-35303bba203b.png">

Reply:
`takes the user back to the start page`

Reply:
Makes sense, I've updated now!